dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -40,6 +40,45 @@
 static HHOOK s_hMouseHook = NULL; // hook handle
 static HWND s_hWatchWnd = NULL; // the window handle to watch
 
+static const LPCWSTR s_prefixes[] =
+{
+    L"file://",
+    L"ftp://",
+    L"http://",
+    L"https://",
+    L"www."
+};
```

Comment:
Aren't these retrieved from the registry? (like, the list of supported protocols)

Reply:
Registry functions are not referred for it.